data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/configuration/AwsConfig.java

```diff
@@ -18,7 +18,7 @@ public static class AwsMskConfig {
         private String arn;
     
         @JsonProperty("broker_connection_type")
-        private MskBrokerConnectionType brokerConnectionType;
+        private MskBrokerConnectionType brokerConnectionType = MskBrokerConnectionType.PUBLIC;
```

Comment:
single VPC connection type should be default. but it can be addressed in next commit